docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+        <p>
+            Add an event listener with the
+            <code>EventTarget.addEventListener()</code> function.
+        </p>
+    </div>
+</section>
+<section>
+    <!-- Keyboard -->
+    <div class="demo" data-dom-diff="typography-keyboard">
+        <p>
+            Press
+            <kbd>Ctrl+Z</kbd> (
+            <kbd>Command+Z</kbd> on Mac) to undo the most recent action.
+        </p>
+    </div>
+</section>
+<section>
+    <!-- Links -->
+    <div class="hxAlert">
+        <div class="hxAlert__icon">
+            <hx-icon type="info-circle"></hx-icon>
+        </div>
+        <div class="hxAlert__text">
+            <span class="hxAlert__status">Note</span>
+            Buttons are better suited to initiate in-page actions.
+        </div>
+    </div>
+    <div class="demo" data-dom-diff="typography-link">
+        <ul>
+            <li>
+                <a href="#">Normal Link</a>
+            </li>
+            <li>
+                <a href="#" class="pseudo-active">Pressed Link</a>
+            </li>
+        </ul>
+    </div>
+</section>
+<section>
+    <!-- Sub-Body -->
+    <div class="demo" data-dom-diff="typography-sub-body">
+        <p class="hxSubBody">Sub-Body</p>
+    </div>
+</section>
+<section>
+    <!-- Subdued -->
+    <div class="demo" data-dom-diff="typography-subdued">
+        <span class="hxSubdued">Subdued</span>
+    </div>
+</section>
+
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-bash">
+        <figcaption>bash Syntax Highlighting Example</figcaption>
+        {% code 'bash' %}
+-        #!/bin/bash
+-
+-        ###### CONFIG
+-        ACCEPTED_HOSTS="/root/.hag_accepted.conf"
+-        BE_VERBOSE=false
+-
+-        if [ "$UID" -ne 0 ]
+-        then
+-          echo "Superuser rights required"
+-          exit 2
+-        fi
+-
+-        genApacheConf(){
+-          echo -e "# Host ${HOME_DIR}$1/$2 :"
+-        }
+-      {% endcode %}
+    </figure>
+</section>
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-css">
+        <figcaption>CSS Syntax Highlighting Example</figcaption>
+        {% code 'css' %}
+-        /* CSS Comment */
+-        @import url('https://fonts.googleapis.com/css?family=Roboto+Mono:100,100i,300,300i,400,400i,700,700i');
+-        :root {
+-          --border-color: #bada55;
+-        }
+-        html {
+-          box-sizing: border-box;
+-        }
+-        *,
+-        *::before,
+-        *::after {
+-          box-sizing: inherit;
+-        }
+-        #navigation {
+-          border: 1px solid var(--border-color);
+-          padding: 1em;
+-          margin: 0;
+-          background-image: linear-gradient(90deg, rgba(0, 0, 0, 0,5), white);
+-        }
+-        input[type="checkbox"]:checked {
+-          /* ... */
+-        }
+-        .text-center {
+-          text-align: center !important;
+-        }
+-      {% endcode %}
+    </figure>
+</section>
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed